sale_partner_approval/data/init_sale_ok.xml

```diff
@@ -0,0 +1,9 @@
+<odoo noupdate="1">
```

Comment:
FMI, wont it be better to use pre init hook? is it still recommended ?

Reply:
This should be equivalent to an init hook, afaics.